docker/Dockerfile

```diff
@@ -57,20 +58,72 @@ ENV LD_LIBRARY_PATH "/usr/local/lib:$LD_LIBRARY_PATH"
 # install KINC
 WORKDIR /opt
 
-ADD https://api.github.com/repos/SystemsGenetics/KINC/git/refs/heads/${KINC_REVISION} kinc-version.json
-RUN git clone -q https://github.com/SystemsGenetics/KINC.git \
-	&& mkdir -p KINC/build \
-	&& cd KINC/build \
-	&& git checkout -q ${KINC_REVISION} \
-	&& qmake ../src/KINC.pro GUI=no \
-	&& make -s -j $(nproc) \
-	&& make -s qmake_all \
-	&& make -s install
+# For older version of KINC we need to use qmake. For newer versions after 3.3.x we
+# can use the make file in the root directory of KINC.
+RUN if [ "${KINC_REVISION}" = "v3.3.x" ] || [ "${KINC_REVISION}" = "v3.2.x" ] ; then \
```

Comment:
I don't think these conditionals will work because KINC_REVISION must be set to a specific version like `v3.3.0` rather than `v3.3.x`, so the docker build might end up running all of these conditionals. You need to check your docker build log and make sure it executed the commands you expected.

Reply:
Perhaps a better way would be to check for the existence of Makefile, because I added the Makefile at the same time I added the R scripts to the build.

Reply:
Yes, good catch. That was my stupid mistake.  I'll fix it.

Reply:
This has been fixed in the most recent push.